res/controllers/Pioneer-DDJ-SX-scripts.js

```diff
@@ -1776,20 +2008,21 @@ PioneerDDJSX.hotCueLeds = function(value, group, control) {
 
 PioneerDDJSX.VuMeterLeds = function(value, group, control) {
     var midiBaseAdress = 0xB0,
-        midiOut = 0;
+        channel = 0x02,
+        midiOut = 0x00;
 
     value = 1 / (1 - PioneerDDJSX.cutVumeter) * (value - PioneerDDJSX.cutVumeter);
     if (value < 0) {
         value = 0;
     }
 
-    value = parseInt(value * 0x77); //full level indicator: 0x7F
+    value = parseInt(value * 0x76); //full level indicator: 0x7F
     if (value < 0) {
         value = 0;
     }
 
-    if (value > 0x77) {
-        value = 0x77;
+    if (value > 0x76) {
+        value = 0x76;
     }
 
     if (engine.getValue(group, "PeakIndicator")) {
```

Comment:
Yes, exactly. This is an effect I realized when switching from "DJ software other than Serato"-mode to "Serato"-mode.